<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteCreatorWidget</class>
 <widget class="QWidget" name="PaletteCreatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="interoplateButton">
     <property name="toolTip">
      <string>Toggle palette value interpolation</string>
     </property>
     <property name="text">
      <string>Interpolate</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PaletteTableView" name="paletteTableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addCtrlPointButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Add new control point</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delCtrlPointButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Delete selecte control point</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaletteTableView</class>
   <extends>QTableView</extends>
   <header>palettetableview.h</header>
   <slots>
    <slot>addControlPoint()</slot>
    <slot>delControlPoint()</slot>
    <slot>toggleInterpolation(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>addCtrlPointButton</sender>
   <signal>released()</signal>
   <receiver>paletteTableView</receiver>
   <slot>addControlPoint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delCtrlPointButton</sender>
   <signal>released()</signal>
   <receiver>paletteTableView</receiver>
   <slot>delControlPoint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>interoplateButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>paletteTableView</receiver>
   <slot>toggleInterpolation(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
